packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardFilterBuilder.tsx

```diff
@@ -297,13 +297,9 @@ export const CloudPulseDashboardFilterBuilder = React.memo(
           <Button
             startIcon={
               showFilter ? (
-                <KeyboardArrowDownIcon
-                  sx={{ color: 'grey', height: '30px', width: '30px' }}
-                />
+                <KeyboardArrowDownIcon />
               ) : (
-                <KeyboardArrowRightIcon
-                  sx={{ color: 'grey', height: '30px', width: '30px' }}
-                />
+                <KeyboardArrowRightIcon />
```

Comment:
since we removed, how it looks like, is there any change in UI

Reply:
no nothing is changed, it is same. This sx was not even applied to that